x/consensus/keeper/concensus_keeper.go

```diff
@@ -410,14 +428,14 @@ func (k Keeper) SetMessagePublicAccessData(
 		ValAddress: valAddr,
 		Data:       msg.GetData(),
 	}
-	err = cq.SetPublicAccessData(ctx, msg.GetMessageID(), payload)
+	err = cq.SetPublicAccessData(sdkCtx, msg.GetMessageID(), payload)
 	if err != nil {
-		k.Logger(ctx).Error("error while adding message public access data", "err", err)
+		k.Logger(sdkCtx).Error("error while adding message public access data", "err", err)
```

Comment:
Replace with liblog usage as above for other touched occurrences in this file and others.

Reply:
updated